pkg/controller/data/default-model-aliases.yaml

```diff
@@ -25,7 +25,7 @@ items:
     spec:
       manifest:
         alias: text-embedding
-        model: text-embedding-3-small
+        model: text-embedding-3-large
```

Comment:
This won't update currently deployed environments, but this is the right change.